README.md

```diff
@@ -1 +1,5 @@
-# cookit-backend
\ No newline at end of file
+# cookit-backend
+Backend Go App contains:
+-RestApi 
+-MongoDB
+-...
```

Comment:
A te magiczne kropki to co? :D Jeśli chcesz to do portfolio mieć, to warto mieć uzupełnione README.md bo to jest pierwszy plik w który każdy patrzy wchodząc w projekt

Reply:
Tak, README.md nie skończyłem właśnie, gdyż to była tzw. wersja "demo". Chciałem się zastanowić, co powinno się w nim zawrzeć, aby przyciągnąć ewentualnych rekruterów. Nie wiedziałem, czy zawrzeć w nim jakieś pakiety np. UUID do generowania unikalnych identyfikatorów, czy validatora, aby pokazać, że takie rzeczy znam na podstawowym poziomie. Oczywiście testy też mam w planach, ale póki co chciałem skończyć aplikację, dlatego ich nie dopisywałem.

Reply:
Co użyte(i może warto napisać dlaczego), jak uruchomić projekt, flagi które są potrzebne do uruchomienia